stdlib/collections/__init__.pyi

```diff
@@ -43,7 +43,9 @@ class UserDict(MutableMapping[_KT, _VT], Generic[_KT, _VT]):
     @overload
     def __init__(self, __dict: SupportsKeysAndGetItem[_KT, _VT], **kwargs: _VT) -> None: ...
```

Comment:
Hm, the same is true here: `SupportsKeysAndGetItem[str, _VT]` is the only possible case where `**kwargs` will be a good fit.